e2e/capacity/transactions.test.ts

```diff
@@ -142,6 +144,37 @@ describe('Capacity Transactions', function () {
           assert.equal(capacityStaked.totalTokensStaked.toBigInt(), stakedForMsa);
           assert.equal(capacityStaked.totalCapacityIssued.toBigInt(), stakedForMsa / getTokenPerCapacity());
         });
+
+        it('successfully pays with Capacity for eligible transaction - addRecoveryCommitment', async function () {
+          // Generate a recovery secret using the Recovery SDK
+          const recoverySecret = generateRecoverySecret();
+
+          // Generate Recovery Commitment using the Recovery SDK with test email contact
+          const testEmail = 'test@example.com';
+          const recoveryCommitmentHex = getRecoveryCommitment(recoverySecret, ContactType.EMAIL, testEmail);
+
+          // Convert hex string to Uint8Array for the payload
+          const recoveryCommitment = new Uint8Array(Buffer.from(recoveryCommitmentHex.slice(2), 'hex'));
```

Comment:
This shouldn't work. It is formatted in `ABCD-1234-...` as user formatted. So the slice would drop data and apparently the buffer doesn't care about the `-`s?